Tests/FunctionalTests/TestApp/Tasks/AzureSDK.js

```diff
@@ -0,0 +1,31 @@
+try {
+    var storage = require("@azure/storage-blob");
```

Comment:
Wrap in try/catch so that functionals can "try" to import this file for node4 and node6